src/main/java/com/mcmiddleearth/guidebook/command/GuidebookOff.java

```diff
@@ -24,6 +25,13 @@ public GuidebookOff(String... permissionNodes) {
     protected void execute(CommandSender cs, String... args) {
         Player player = (Player) cs;
         PluginData.exclude(player);
+
+        for (InfoArea area : PluginData.getInfoAreas().values()) {
+            if (area.isInformed(player)) {
+                area.removeInformedPlayer(player);

```

Comment:
With the change I made to return early if the player has disabled Guidebook, it meant the following was possible 

Players turns off Guidebook whilst in a region, leaves region, turns Guidebook back on and then re-enters that same region -> player doesn't receive the title popup

So I added this to un-inform them